protocol/x/clob/keeper/clob_pair.go

```diff
@@ -304,6 +304,42 @@ func (k Keeper) GetAllClobPairs(ctx sdk.Context) (list []types.ClobPair) {
 	return
 }
 
+// validateLiquidationAgainstClobPairStatus returns an error if placing the provided
+// liquidation order would conflict with the clob pair's current status.
+func (k Keeper) validateLiquidationAgainstClobPairStatus(
```

Comment:
Question: should we unit test this function?

Reply:
adding test cases to PlacePerpetualLiquidation for this